crypto/fipsmodule/rsa/rsa.c

```diff
@@ -470,7 +602,8 @@ int RSA_sign_raw(RSA *rsa, size_t *out_len, uint8_t *out, size_t max_out,
 
 unsigned RSA_size(const RSA *rsa) {
   SET_DIT_AUTO_DISABLE;
-  size_t ret = rsa->meth->size ? rsa->meth->size(rsa) : rsa_default_size(rsa);
+  size_t ret = (rsa->meth && rsa->meth->size) ?
```

Comment:
I suppose we should allow the SEGFAULT if `rsa` is NULL?

Reply:
Ahhh sorry I guess I can expand the check. A consumer shouldn't really be calling this on a NULL key but I guess the less jagged edges the better.